pubspec.yaml

```diff
@@ -10,17 +10,18 @@ dependencies:
   flutter:
     sdk: flutter
 
-  cloud_firestore: ^4.2.0
-  cloud_functions: ^4.0.6
-  fake_cloud_firestore: ^2.1.0
-  firebase_analytics: ^10.0.7
-  firebase_auth: ^4.2.1
-  firebase_core: ^2.4.0
-  firebase_crashlytics: ^3.0.7
-  firebase_dynamic_links: ^5.0.9
-  firebase_messaging: ^14.1.4
-  firebase_remote_config: ^3.0.7
-  firebase_storage: ^11.0.8
+  cloud_firestore: ^4.9.1
+  cloud_functions: ^4.4.0
+  fake_cloud_firestore: ^2.4.1+1
+  firebase_analytics: ^10.4.5
+  firebase_auth: ^4.9.0
+  firebase_core: ^2.15.1
+  firebase_crashlytics: ^3.3.5
+  firebase_dynamic_links: ^5.3.5
+  firebase_messaging: ^14.6.7
+  firebase_remote_config: ^4.2.5
+  firebase_storage: ^11.2.6
+  cloud_firestore_platform_interface: ^5.16.0
```

Comment:
Why do need a `UNLISTED` package i.e `cloud_firestore_platform_interface` ?

Reply:
Please remove it

Reply:
Removed.